src/main/java/org/folio/search/utils/SearchUtils.java

```diff
@@ -200,4 +201,19 @@ public static String getEffectiveCallNumber(String prefix, String callNumber, St
     return fulltextValue;
   }
 
+  /**
+   * Checks if passed value null or empty collection or map.
+   *
+   * @param object - value to check
+   * @return true if null or empty, false - otherwise.
+   */
+  public static boolean isNotNullOrEmpty(Object object) {
```

Comment:
Is it possible to use Apache commons for such kind of checks?

Reply:
resolved, I've replaced it with `ObjectUtils.isNotEmpty(Object value)`
thanks for noticing this!